content/php/concepts/string-functions/terms/str-repeat/str-repeat

```diff
@@ -0,0 +1,56 @@
+---
+Title: 'str_replace()'
+Description: 'Returns a string with occurrences of a specified substring replaced by another string.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+---
+Title: 'str_replace()'
+Description: 'Returns a string with occurrences of a specified substring replaced by another string.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+# str_repeat()
```

Comment:
Delete this header. The metadata Title key will provide the title for the entry. Once you delete this and the repeated metadata, make sure there's only one blank line between the end of the metadata and the first line of the entry.